ffi/capi/src/locale_core.rs

```diff
@@ -101,7 +102,7 @@ pub mod ffi {
         /// Set the region part of the [`Locale`].
         #[diplomat::rust_link(icu::locale::Locale::try_from_str, FnInStruct)]
         #[diplomat::rust_link(icu::locale::Locale::try_from_utf8, FnInStruct, hidden)]
-        #[diplomat::attr(all(*, not(dart)), setter = "region")]
+        #[diplomat::attr(auto, setter = "region")]
```

Comment:
issue: this should be `all(auto, not(dart))`. and below

Reply:
I see, thank you! Fixing now